### PR TITLE
WIP: Add npm package with binwrap

### PR DESCRIPTION
**Pull request by avh4**
Once the binaries are published to some location and the URLs in `index.js` set properly, then `npm test` will validate the the binaries can be downloaded, then `npm publish` should work.

In elm-format, I put all this stuff in `package/npm/*` instead of at the project root, which I thought was a bit cleaner, but it a bit unusual.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "elmi-to-json",
```
Comment (avh4) [inline package.json]: ❓ will the name be `elmi-to-json`, or `elm-interface-to-json`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "version": "0.19.0",
```
Comment (avh4) [inline package.json]: `elm-test` follows this convention for version numbers of having the `0.19` match the Elm version it works with, and the patch version just increments.  That has seemed to work fine for elm-test, though I use a more normal versioning scheme for elm-format, and also, I think elmi-to-json will only be used by advanced users, so the versioning scheme probably doesn't matter that much 🤷‍♀️

File: package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "bin/elmi-to-json"
+  ],
+  "bin": {
+    "elmi-to-json": "bin/elmi-to-json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stoeffel/elmi-to-json.git"
+  },
+  "author": "",
```
Comment (avh4) [inline package.json]: description and author probably need to be filled in before npm will allow it to be published.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/stoeffel/elmi-to-json/issues"
+  },
+  "homepage": "https://github.com/stoeffel/elmi-to-json#readme",
+  "dependencies": {
+    "binwrap": "^0.2.0-rc1"
```
Comment (avh4) [inline package.json]: If everything seems to work, I will promote this to 0.2.0.

File: package.json (hunk 1 above)
Comment (stoeffel) [reply package.json]: I would like to use `elmi-to-json`

File: package.json (hunk 2 above)
Comment (stoeffel) [reply package.json]: sgtm

**Pull request merged**